python/ray/tests/test_client_terminate.py

```diff
@@ -113,11 +113,12 @@ def f():
         class MetadataIsCorrectlyPassedException(Exception):
             pass
 
-        def mock_terminate(term, metadata):
-            raise MetadataIsCorrectlyPassedException(metadata[1][0])
+        def mock_terminate(self, term):
```

Comment:
what term is used for?

Reply:
This mocks the `Terminate()` handler, which accepts the `TerminateRequest` (`term`). We need the same function signature.